docker/Makefile

```diff
@@ -35,7 +35,8 @@ unit-tests-ci:
 .PHONY: build
 build:
 	npm install
-	docker-compose build mail app ld-api next-proxy next-app
+    # TODO: When this Makefile is moved to repo root, turn "../" below into "./"
+	LFMERGE_VERSION="$(shell jq -r .lfmerge ../other-dependencies.json)" docker-compose build mail app ld-api next-proxy next-app
```

Comment:
`shell` nor `jq` can be assumed to be present on the dev's machine unless we add instructions to the README about additional local installs which I'd prefer not to do.  Let's discuss this approach further.

Reply:
`shell` is a Makefile keyword, but you're right about `jq` needing to be added to the README.

Reply:
I was thinking about this too.  We have several common *nix tools that we are using that have snuck in.  sed is also on the list and it's been in our scripts for a while.  

I think while it would have been nice to have a pure Windows dev on the team, both Hanna and Joseph were keen to use WSL Ubuntu and so we actually only support Linux and MacOS dev environments at this point.

Our small list of dev dependencies is now:
Docker
make
sed
bash
jq

Anything else?

Reply:
I had another thought that would eliminate the need for `jq`, I just don't want to keep adding install requirements...it just creates more maintenance issues.

What if we did something like this.

`docker/lfmerge/Dockerfile`:
```
FROM ghcr.io/sillsdev/lfmerge:2.0.98
```

That will eliminate the `.json` and the need for additional installs.  Grabbing the version now can use `head` and `cut`.  The `docker-compose.yml` would just need a reference to the `Dockerfile` like many of our other containers.

Reply:
FYI, `head` and `cut` are both available in our GHA runners: https://github.com/longrunningprocess/gha-sandbox/runs/6064688705?check_suite_focus=true

Reply:
Done in commits 02b2456 and e508b93.